cmds/core/find/find.go

```diff
@@ -18,21 +18,46 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"strings"
 
 	"github.com/u-root/u-root/pkg/find"
 )
 
+type params struct {
+	fileType string
+	name     string
+	perm     int
+	long     bool
+	debug    bool
+}
+
+type command struct {
```

Comment:
I have a thought here. We could follow the pattern of exec/command, and call this
cmd
and call the function that creates it
command

the parallel structure with the os/exec package, in terms of naming, might be helpful?

Reply:
Yeah, I think it's a good idea.